### PR TITLE
Dockerise this service

### PR DESCRIPTION
**Pull request by baely**
containerises the devhou.se website. also clean up some stuff re: mistranslated base pages and removing references to devhouse shop

two stage docker:
  - hugo to generate the static website
  - nginx basic web server to serve the files

some other auxiliary files to support

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    external: true
+
+services:
+  web:
+    image: registry.baileys.dev/devhouse:latest
+    build:
+      context: .
+      dockerfile: Dockerfile
+    networks:
+      - cowardscloud_web
```
Comment (dbut2) [inline docker-compose.yaml:13]: is this attachable for external services?

Comment (dbut2) [reply docker-compose.yaml:13]:
```
dylan@dylans-node-dev ~ 
> $ docker network inspect cowardscloud_web -f "{{ .Attachable }}"
false
```

Comment (baely) [reply docker-compose.yaml:13]: this isn't an external service. it lives on the cowards cloud

Comment (dbut2) [reply docker-compose.yaml:13]: it's external to the stack

Comment (baely) [reply docker-compose.yaml:13]: "the stack"? it is it's own stack defined in the docker-compose.yaml. you can mount any external overlay network

**Pull request merged**